app/views/miq_request/_prov_dialog_volume_fieldset.html.haml

```diff
@@ -30,7 +30,7 @@
                                                       "data-trigger"      => "focus",
                                                       "data-placement"    => "top",
                                                       "data-toggle"       => "popover",
-                                                      "data-content"      => @record.try(:decorate).try(:provisioning_volume_size_tooltip)}
+                                                      "data-content"      => @record.try(:decorate).try(:provisioning_volume_size_tooltip).nil? ? "Default value is 1" : @record.try(:decorate).try(:provisioning_volume_size_tooltip)}
```

Comment:
Here, we do not need to check for for nil? , i think we need to check whether that field exists or not, if it does not exists, then show default one. Can you check if this code works?
```
@record.try(:decorate).try(:provisioning_volume_size_tooltip) ? @record.try(:decorate).try(:provisioning_volume_size_tooltip)} : _("Default value is 1")